lib/liquid.rb

```diff
@@ -20,31 +22,31 @@
 # WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 module Liquid
-  FilterSeparator             = /\|/
-  ArgumentSeparator           = ','.freeze
-  FilterArgumentSeparator     = ':'.freeze
-  VariableAttributeSeparator  = '.'.freeze
-  WhitespaceControl           = '-'.freeze
-  TagStart                    = /\{\%/
-  TagEnd                      = /\%\}/
-  VariableSignature           = /\(?[\w\-\.\[\]]\)?/
-  VariableSegment             = /[\w\-]/
-  VariableStart               = /\{\{/
-  VariableEnd                 = /\}\}/
-  VariableIncompleteEnd       = /\}\}?/
-  QuotedString                = /"[^"]*"|'[^']*'/
-  QuotedFragment              = /#{QuotedString}|(?:[^\s,\|'"]|#{QuotedString})+/o
-  TagAttributes               = /(\w+)\s*\:\s*(#{QuotedFragment})/o
-  AnyStartingTag              = /#{TagStart}|#{VariableStart}/o
-  PartialTemplateParser       = /#{TagStart}.*?#{TagEnd}|#{VariableStart}.*?#{VariableIncompleteEnd}/om
-  TemplateParser              = /(#{PartialTemplateParser}|#{AnyStartingTag})/om
-  VariableParser              = /\[[^\]]+\]|#{VariableSegment}+\??/o
+  FILTER_SEPARATOR             = /\|/.freeze
+  ARGUMENT_SEPARATOR           = ','
+  FILTER_ARGUMENT_SEPARATOR     = ':'
+  VARIABLE_ATTRIBUTE_SEPARATOR  = '.'
+  WHITESPACE_CONTROL = '-'
+  TAG_START = /\{\%/.freeze
+  TAG_END                      = /\%\}/.freeze
+  VARIABLE_SIGNATURE           = /\(?[\w\-\.\[\]]\)?/.freeze
+  VARIABLE_SEGMENT             = /[\w\-]/.freeze
+  VARIABLE_START               = /\{\{/.freeze
+  VARIABLE_END                 = /\}\}/.freeze
+  VARIABLE_INCOMPLETE_END = /\}\}?/.freeze
```

Comment:
how come this isn't indented like the previous and the next line?

Reply:
I didn't notice this one, I'll look in to it as all the rest collapsed to a single space